objectRecognitionSource/src/test/java/io/vantiq/extsrc/objectRecognition/neuralNet/TestNoProcessorQueries.java

```diff
@@ -48,12 +48,15 @@ public static void tearDown() {
             core.stop();
             core = null;
         }
-        deleteSource();
-        deleteDirectory(OUTPUT_DIR);
-        
-        for (int i = 0; i < vantiqUploadFiles.size(); i++) {
-            deleteFileFromVantiq(vantiqUploadFiles.get(i));
+        if (vantiq != null && vantiq.isAuthenticated()) {
```

Comment:
@rblumer4 This sider test appears to be buggy -- it complains about all the `@AfterClass` annotations.  It says I can _close the issue_ but when I try, it says I don't have permission.  Not a big deal, but it would be nice to be able to close these -- which, presumably would shut it up for the duration of the PR.

Not a big deal & no rush.

Reply:
may need to create a `sider.yml` file in this repository then, and turn off that check.
Per this PR:  https://github.com/Vantiq/vantiq-extension-sources/pull/240

Or we can turn Sider off for this repo entirely.

Reply:
Turns out (once I found my way), that I can  @SuppressWarnings("PMD.JUnit4TestShouldUseAfterAnnotation") 
on the method.  So for the various annoying ones, this works.

This particular thing is a left-over from the JUnit 3 -> JUnit4 transition that doesn't affect us.  So I just suppressed the warning locally -- not too many of them...